### DISCUSSION TITLE
Filtering org-level issue templates in repositories

### DISCUSSION THREAD

**Discussion by Daeraxa** [community/community#38435] [Projects and Issues]
If I have a bunch of different repositories - some website based, some application based, some plugins etc. all under the same organisation, is there any way to filter which issue or pull request templates are actually displayed?

For example if I'm logging an issue for one of the website repos I only want the ISSUE_WEBSITE.yml template to show up, if I'm logging an issue for the core application I might want both the ISSUE_APPLICATION.yml and ISSUE_PLUGIN.yml templates to appear... not to mention different forms for feature requests.

I would want to keep the org level templates as there are a *lot* of repos which would make editing each one a bit of a nightmare, just need to be able to filter them out.

**Comment by david-engelmann** [comment]
[This is an active issue in the actions repo, check it out!](https://github.com/actions/toolkit/issues/1054) It would a great feature
